Cargo.toml

```diff
@@ -241,6 +241,7 @@ objc2-ui-kit = { version = "0.3.1", default-features = false, features = [
 
 # Windows
 [target.'cfg(target_os = "windows")'.dependencies]
+scopeguard = "1.2.0"
```

Comment:
I'd prefer a helper struct with a custom `Drop` impl that calls `RegCloseKey` over the added dependency.